### PR TITLE
Gibs squelch when walked over

### PR DESCRIPTION
**Pull request by cacogen**
:cl: cacogen
add: Gibs squelch when walked over
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

This makes them more visceral. Plus more sound effects are always good.

[gib_step.ogg](https://clyp.it/bymg2kdn)

### CODE REVIEW

**Comment by JJRcop** [comment]
Can you source the sound effect?

**Comment by MrDoomBringer** [comment]
good PR

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 1]
```diff
@@ -56,6 +56,14 @@
 /obj/effect/decal/cleanable/blood/gibs/ex_act(severity, target)
 	return
 
+/obj/effect/decal/cleanable/blood/gibs/Crossed(mob/living/L)
+	if(istype(L) && has_gravity(loc))
+		if(L.has_trait(TRAIT_LIGHT_STEP))
+			playsound(loc, 'sound/effects/gib_step.ogg', 20, 1)
```
Comment (Fox-McCloud) [inline code/game/objects/effects/decals/cleanable/humans.dm]: `playsound(loc, 'sound/effects/gib_step.ogg', L.has_trait(TRAIT_LIGHT_STEP) ? 20 : 50, 1)`

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 2]
```diff
@@ -56,6 +56,11 @@
 /obj/effect/decal/cleanable/blood/gibs/ex_act(severity, target)
 	return
 
+/obj/effect/decal/cleanable/blood/gibs/Crossed(mob/living/L)
+	if(istype(L) && has_gravity(loc))
+		playsound(loc, 'sound/effects/gib_step.ogg', L.has_trait(TRAIT_LIGHT_STEP) ? 20 : 50, 1)
+	. = ..()
```
Comment (ninjanomnom) [inline code/game/objects/effects/decals/cleanable/humans.dm]: `return ..()`

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by Jalleo — CHANGES_REQUESTED** [review]
Under the licence that you have linked through that website it says we need to link to the person in the way they wanted and as such how do they want their attribution to be known.

**Comment by Shadowflame909** [comment]
Cacogen would like there attribution to be known by re-enabling coder credits and making the entire serverbase angry

**Comment by cacogen** [comment]
@Jalleo 

There's a link to that sound file in the .ogg, is that good enough? Otherwise I'll have to find another file.

**Pull request merged**